action.yml

```diff
@@ -18,5 +18,5 @@ inputs:
     required: false
     default: "UTC"
 runs:
-  using: "node12"
+  using: "node16"
```

Comment:
Funnily enough, CI was already running Node 16.

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime